xml/dns_versionbind.xml

```diff
@@ -8,6 +8,40 @@
     dnsmasq-2.76-1-ubnt2
   -->
 
+  <!--
+  The following 'assert nothing' block is intended to handle banners so simple
+  that they cannot be attributed to a product or vendor. They are at the
+  beginning of the file as a performance tweak given how frequenty they occur.
+  -->
+
+  <fingerprint pattern="^none$">
+    <description>bare 'none' -- assert nothing.</description>
+    <example>none</example>
+    <param pos="0" name="service.certainty" value="0.0"/>
+    <param pos="0" name="service.protocol" value=""/>
+  </fingerprint>
+
+  <fingerprint pattern="(?i)^unknown$">
+    <description>bare 'unknown' -- assert nothing.</description>
+    <example>unknown</example>
+    <param pos="0" name="service.certainty" value="0.0"/>
+    <param pos="0" name="service.protocol" value=""/>
+  </fingerprint>
+
+  <fingerprint pattern="^no version$">
+    <description>bare 'no version' -- assert nothing.</description>
+    <example>no version</example>
+    <param pos="0" name="service.certainty" value="0.0"/>
+    <param pos="0" name="service.protocol" value=""/>
+  </fingerprint>
+
+  <fingerprint pattern="^$">
```

Comment:
move this to the top? debatable, as project sonar has a ton of empty replies on version.bind, but other commercial products may have non-empty replies first, in which case this move is not a real optimization for them. 

On the other hand, it's just a single, simple regular expression...

Reply:
Actually, no, I should have done that.  It's a high volume match and computationally cheap. Thanks.